src/service/owners-impl.js

```diff
@@ -15,12 +15,28 @@
  */
 
 /* eslint-disable no-unused-vars */
+import {Resource, ResourceState} from './resource';
```

Comment:
eventually, we'd remove all the dependencies on `resource.js`.

Please leave a TODO: 
- Resource.setOwner, Resource.getOwner should be moved here.
- ResourceState.NOT_BUILT might not be needed here.